parity/rpc.rs

```diff
@@ -253,7 +255,15 @@ pub fn new_ipc<D: rpc_apis::Dependencies>(
 		}
 	}
 
-	match rpc::start_ipc(&conf.socket_addr, handler, rpc::RpcExtractor) {
+	// some validations ..
+	let chmod = conf.chmod;
+	if chmod.len() != 3 && chmod.len() != 4 {
```

Comment:
nitpick: according to the man pages: `A numeric mode is from one to four octal digits (0-7)`

```suggestion
	if chmod.len() >= 1 && chmod.len() <= 4 {
```